gradoop-temporal/src/main/java/org/gradoop/temporal/model/api/TimeDimension.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.temporal.model.api;
+
+/**
+ * Attributes accessible on {@link org.gradoop.temporal.model.impl.pojo.TemporalElement}s representing
+ * different time dimensions.
+ * For each dimension, two fields are given to represent a time interval.
+ */
+public enum TimeDimension {
```

Comment:
This class is already part of `gradoop-temporal`, seems like your branch was created before that module was merged. Please rebase your commits to `develop`